roles/openshift_logging/files/generate-jks.sh

```diff
@@ -73,6 +73,7 @@ function generate_JKS_chain() {
         -keystore $dir/$NODE_NAME.jks \
         -storepass $ks_pass \
         -noprompt \
+        -trustcacerts \
```

Comment:
Correct, adding -trustcacerts will just allow for "system CA" to be considered trusted when using this JKS store! The "system ca's" are stored in a certificates file named "cacerts" resides in the security properties directory, java.home/lib/security (and is a JKS store its self). 

This means, that IM CA's not added to that keystore, will not be trusted, so while allowing -trustcacerts is not a bad thing to have in this case, its not helping to solve the real problem. 

That said, related to this ansible file. There is all kinda of wrong with this script! 

1. Why are we using it an not using https://docs.openshift.com/container-platform/3.5/dev_guide/secrets.html#service-serving-certificate-secrets for the default scripts? 
2. Why does our method of setting up the CA (for a java application) differ from the process that MW uses with its templates (they require that you provide a JKS in a secret). Why does out image not require that, and if one is not provided, why don't we refer back to 1 or generate our own JKS in the image we ship? 

Doing thing in Ansible like this (and not using a common library) means that we end up having (by my count 3 ) different ways to setup / provide certificates to various parts of the system.